lib/widgets/shipanther_scaffold.dart

```diff
@@ -43,7 +44,40 @@ class ShipantherScaffold extends StatelessWidget {
           // Important: Remove any padding from the ListView.
           padding: EdgeInsets.zero,
           children: <Widget>[
-                _createHeader(user),
+                // _createHeader(),
+                UserAccountsDrawerHeader(
```

Comment:
This should go in `_createHeader`. It is important to keep changes from main when rebasing and add on top of it.